sshfs.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: doc
+title: SSHFS Workflow Tutorial
+---
+
+## Why SSHFS?
+
+SSHFS is a tool that allows you to mount any directory from a remote machine on your local machine, and consequently use your favorite text editor/IDE/code editing environment and still be able to compile and run your code on the remote machine.
+
+## Installation
+
+You need to install SSHFS on your own machine (Windows, Linux, or MacOS).
+
+### Ubuntu/Debian
+
+You can easily install SSHFS using apt-get as follows:
+
+```
+sudo apt-get install sshfs
+```
+
+### MacOS
+
+On MacOS, you can install SSHFS via FUSE and SSHFS from the [osxfuse site.](https://osxfuse.github.io). You will want to download both packages. 
+
+Alternativey, if you have [Homebrew](http://brew.sh) installed, you can also install it using these two commands:
+
+```
+brew cask install osxfuse           # installs osxfuse
+brew install homebrew/fuse/sshfs    # installs sshfs
+```
+
+### Windows
+
+For Windows, you can either install it in the Ubuntu subsystem using apt-get, or download the [win-sshfs](https://win-sshfs.googlecode.com/files/win-sshfs-0.0.1.5-setup.exe) package. 
+
+## Using SSHFS
+
+If you are trying to mount your CS241 virtual machine, you **do** need to be on the campus network or VPN to be able to connect. These instructions apply to Linux and MacOS. Windows will be later.
+
+### Mounting the Remote Filesystem
+
+You'll want to make a directory to mount the remote file system on. Basically, sshfs will make it a "portal" from your machine to the remote machine. The directory you make will be used in the next step.
+
+```
+mkdir ~/remote
+```
+
+Then, you can use sshfs to mount the file system locally. You can also specify what the entry point is on the remote machine, rather than just starting at the root of the drive (/), you can have it start in your home folder (~) or any folder, really. replace <MOUNT_POINT> with the entry point you would like, or leave it empty to mount the root of the drive.
```

Comment:
Explain what the mount point means. Also explain if you can have anything else in the folder. To the staff, this is all easy. But the students who are interested don't even know what a file system is at this point, so mounting is beyond them.